src/app/components/score-chart/score-chart.component.ts

```diff
@@ -1,19 +1,64 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
 import * as Highcharts from 'highcharts';
+import more from 'highcharts/highcharts-more';
+import { parseISO } from 'date-fns';
+import { mean } from 'lodash';
+
+more(Highcharts);
 
 @Component({
   selector: 'app-score-chart',
   templateUrl: './score-chart.component.html',
   styleUrls: ['./score-chart.component.sass']
 })
-export class ScoreChartComponent implements OnInit {
+export class ScoreChartComponent implements OnChanges {
 
-  private options: any = {};
+  @Input()
+  public historicalData: string[];
 
-  constructor() { }
+  @Input()
+  public label: string;
 
-  ngOnInit(): void {
-    Highcharts.chart('container', this.options);
-  }
+  @Input()
+  public currentValue = '- -';
+
+  public averageValue = '--';
+  public Highcharts: typeof Highcharts = Highcharts;
+  public chartOptions: Highcharts.Options = {
+    title: {
+      text: 'Track Your Progress'
+    },
+
+    yAxis: {
+        title: {
+            text: 'Score'
```

Comment:
Not sure what score is this (accent? clarity?)

Reply:
The label is used to show whether it is accent or clarity score. I can combine this with label as well to make it clearer

Reply:
Yep, adding it in the graph (either title or axis or legend) would make it clearer.